servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NativeTransportUtils.java

```diff
@@ -60,9 +63,19 @@ private NativeTransportUtils() {
     }
 
     private static void logUnavailability(final String transport, final String os, final Throwable cause) {
+        if (NETTY_NO_NATIVE) {
+            LOGGER.warn("io.netty:netty-transport-native-{} is explicitly disabled with \"-D{}=true\". Note that it " +
```

Comment:
`LOGGER.warn` -> should this be `info` or something lower as they are explicitly opting in?